test/acceptance/SeleniumSpec.scala

```diff
@@ -0,0 +1,456 @@
+
+    assume(Dependencies.IdentityFrontend.isAvailable,
+      s"- ${Dependencies.IdentityFrontend.url} unavaliable! " +
+        "\nPlease run identity-frontend server before running tests.")
+  }
+
+  /* Register a new Identity user */
+  def withRegisteredIdentityUserFixture(testFun: TestUser => Any) {
+    checkDependenciesAreAvailable
+
+    val testUser = new TestUser
+
+    // When Users visit 'Identity Register' page
+    val register = new pages.Register(testUser)
+    go.to(register)
+    assert(register.hasLoaded())
+
+    // And they fill in personal details
+    register.fillInPersonalDetails()
+
+    // And they submit the form to create Identity account
+    register.createAccount()
+    assert(register.hasCreated())
+
+    testFun(testUser)
+  }
+
+  feature("Subscriptions checkouts") {
+
+    scenario("Guest user subscribes for the Digital Pack with direct debit", AcceptanceTest) {
+      checkDependenciesAreAvailable
+      val testUser = new TestUser
+
+      val checkout = Checkout(testUser)
+      When("Users visit 'Checkout' page ")
+      go.to(checkout)
+      assert(checkout.pageHasLoaded())
+
+      Then("section 'Your details' should load.")
+      assert(checkout.yourDetailsSectionHasLoaded())
+
+      When("they fill in personal details,")
+      checkout.fillInPersonalDetails()
+
+      And("click on 'Continue' button,")
+      checkout.clickPersonalDetailsContinueButton()
+
+      Then("section 'Billing Address' should load.")
+      assert(checkout.billingAddressSectionHasLoaded())
+
+      When("they fill in their billing address")
+      checkout.fillInBillingAddress()
+
+      And("click on 'Continue' button,")
+      checkout.clickBillingDetailsContinueButton()
+
+      Then("section 'Payment Details' should load.")
+      assert(checkout.directDebitSectionHasLoaded())
+
+      When("they fill in direct debit payment details")
+      checkout.fillInDirectDebitPaymentDetails()
+
+      And("select 'Confirm account holder' checkbox,")
+      checkout.selectConfirmAccountHolder()
+
+      And("click on 'Continue' button,")
+      checkout.clickDebitPaymentContinueButton()
+
+      Then("section 'Confirm and Review' should load.")
+      assert(checkout.reviewSectionHasLoaded())
+
+      When("they submit the form")
+      checkout.submitPayment()
+
+      Then("they should land on 'Thank You' page.")
+      val thankYou = ThankYou(testUser)
+      assert(thankYou.pageHasLoaded())
+
+      When("they set a password for their account,")
+      thankYou.setPassword("supers4af3passw0rd")
+
+      Then("they should see 'My Profile' button,")
+      assert(thankYou.hasMyProfileButton)
+
+      And("should have Identity cookies.")
+      Seq("GU_U", "SC_GU_U", "SC_GU_LA").foreach { idCookie =>
+        assert(Driver.cookiesSet.map(_.getName).contains(idCookie))
+      }
+    }
+
+    ignore("Identity user subscribes to the Voucher Everyday package with direct debit", AcceptanceTest) {
```

Comment:
This has been ignored for some time. I'm going to investigate and kill/revive it in a separate PR.